examples/demoes/09/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/sciter-sdk/go-sciter"
+	"github.com/sciter-sdk/go-sciter/window"
+)
+
+func main() {
+	win, _ := window.New(sciter.DefaultWindowCreateFlag, sciter.DefaultRect)
+	win.AddQuitMenu()
```

Comment:
Perhaps, it will be better to add some explanation here.

Reply:
Yes, 
I have added details about AddQuitMenu() and window. Code updated  with required explanation.

Reply:
// AddQuitMenu function adds Quit menu (in menubar) and support for
	// CMD+q to terminate / close application on MacOS
	//
	// Note : AddQuitMenu() only works for MacOS on other platform is
	//        does not have any effect.